storage-plans.yaml

```
@@ -1,50 +1,129 @@
 # storage t-shirt sizes
 
 redis:
+  # 568MB RAM, single node, no failover.
   micro:
     replicas: 0
     instance: 'cache.t4g.micro'
 
+  # 568MB RAM, highly-available.
+  micro-ha:
+    replicas: 2
+    instance: 'cache.t4g.micro'
+
+  # 1.5GB RAM, single node, no failover.
+  tiny:
+    replicas: 0
+    instance: 'cache.t4g.micro'
+
+  # 1.5GB RAM, highly-available
+  tiny-ha:
+    replicas: 2
+    instance: 'cache.t4g.micro'
+
+  # 3GB RAM, single node, no failover.
   small:
     replicas: 0
-    instance: 'cache.t4g.small'
+    instance: 'cache.t4g.medium'
 
+  # 3GB RAM, highly-available
   small-ha:
     replicas: 2
-    instance: 'cache.t4g.small'
+    instance: 'cache.t4g.medium'
 
+  # 6.37GB RAM, single node, no failover.
   medium:
-    replicas: 2
-    instance: 'cache.t4g.medium'
+    replicas: 0
+    instance: 'cache.m6g.large'
 
+  # 6.37GB RAM, highly-available
   medium-ha:
     replicas: 2
-    instance: 'cache.t4g.medium'
+    instance: 'cache.m6g.large'
+
+  # 12.93GB RAM, single node, no failover.
+  large:
+    replicas: 0
+    instance: 'cache.m6g.large'
+
+  # 12.93GB RAM, highly-available
+  large-ha:
+    replicas: 2
+    instance: 'cache.m6g.large'
```

These are `large`, so the instance should be larger by a factor of 1. They are currently the same as `medium`.

```suggestion
  large:
    replicas: 0
    instance: 'cache.m6g.xlarge'

  # 12.93GB RAM, highly-available
  large-ha:
    replicas: 2
    instance: 'cache.m6g.xlarge'
```